compiler/damlc/tests/daml-test-files/Set.daml

```diff
@@ -1,10 +1,10 @@
 -- Copyright (c) 2020, Digital Asset (Switzerland) GmbH and/or its affiliates.
 -- All rights reserved.
 
--- @INFO Use infix
--- @INFO Use infix
--- @INFO Use infix
--- @INFO Use infix
+-- @INFO range=57:30-57:77; Use infix
```

Comment:
These can also be addressed.

Reply:
As above. The plan is to do this in stages. First get everything ship-shape and then look for diagnostics that can be removed.

Reply:
fair enough